highlight-model/secrets.py

```diff
@@ -0,0 +1,4 @@
+user = 'admin'
+password = 'KTNa4Z4aCj5D3Qbx'
```

Comment:
shuuuuuu!🤫 @Eldrick19

Reply:
Oof mb, removed it

Reply:
Should we change the password to be safe? this is a public repo so someone could see this